rubocop-rails_config.gemspec

```diff
@@ -12,8 +12,10 @@ Gem::Specification.new do |spec|
   spec.license               = "MIT"
   spec.required_ruby_version = ">= 2.3.0"
 
-  spec.add_dependency "rubocop", "~> 0.60"
+  spec.add_dependency "rubocop", "~> 0.68"
```

Comment:
RuboCop Rails 2.0 requires RuboCop >= 0.70.
https://rubygems.org/gems/rubocop-rails/versions/2.0.1

RuboCop 0.68 and 0.69 will not be required. So I think it is better to require RuboCop 0.70 or higher.

```diff
- spec.add_dependency "rubocop", "~> 0.68"
+ spec.add_dependency "rubocop", "~> 0.70"
```

RuboCop 0.68 and 0.69 can also be dropped from .travis.yml and Appraisal.

Cc @toshimaru

Reply:
Done!